### ISSUE TITLE
[Dec 18 pub] Steps on topic pages on Canada.ca

### ISSUE THREAD

**Issue by pcwsmith** [canada-ca/canada-ca.github.io#21]
Translation received - final edits
@Borges-Porteous

Eng draft: canada-ca.github.io/_posts/2017-08-09-evidence-based-template-designs.markdown
Fre draft: canada-ca.github.io/_posts/2017-08-09-conception-de-modèles-basé-sur-des-preuves.markdown

**Comment by pcwsmith** [comment]
@patlaj @lisafast - this feels about ready to go, do you agree?

**Comment by lisafast** [comment]
Yup but it needs the link fixed to the Starting a business one. Working on it.
@pcwsmith What do you think - if there is no French yet for the Starting a Business one,  do we take the link out of the English version of Steps until there is one?

**Comment by MaryBethBaker** [comment]
@pcwsmith @lisafast @patlaj Any one think they'd like to use Prose for this blog? We can authorize it and it can give you a GUI for drafting posts.

**Comment by pcwsmith** [comment]
to answer your Q @lisafast, posts should only ever be published simultaneously in E and F b/c OL policy. Anything not yet available in the other official language needs to stay in the drafts folder or be marked published: false if in the posts folder. So no links to posts until both langs are there!

**Comment by pcwsmith** [comment]
@MaryBethBaker I've not looked into Prose, but willing to give it a whirl. Does it work in the github web interface?

**Comment by pcwsmith** [comment]
NB this still needs a plain language / style guide edit. Once this is done, the translation will need to be updated to match (oopsie, still ironing out the process!)
